frontend/src/core/loaders/components/SkeletonLoader.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './SkeletonLoader.css';
+
+let list;
+
+export default function SkeletonLoader (props) {
+    { list = props.entities.length === 0 ? [...Array(30).keys()] : [...Array(2).keys()] }
```

Comment:
I would slice this into a few variables, which will be used later on:

```js
const firstLoad = props.entities.length === 0;
const entityCount = firstLoad ? 30 : 2;
const list = [...Array(entityCount).keys()];
```

You can also drop the `{}` and `let list` above;